server/src/main/java/org/elasticsearch/action/fieldcaps/TransportFieldCapabilitiesAction.java

```diff
@@ -75,8 +73,6 @@ public TransportFieldCapabilitiesAction(
         this.clusterService = clusterService;
         this.indexNameExpressionResolver = indexNameExpressionResolver;
         this.indicesService = indicesService;
-        final Set<String> metadataFields = indicesService.getAllMetadataFields();
```

Comment:
I think we can also remove IndicesService.getAllMetadataFields() now as well?

Reply:
And indeed MapperRegistry.getAllMetadataMapperParsers()

Reply:
Yes, I removed them in https://github.com/elastic/elasticsearch/pull/86522/commits/202f5a88f8d6cdbe07d165b6d0fecb11fc1bc7ee.